NYTimes.com.js

```diff
@@ -165,8 +166,8 @@ function scrape(doc, url) {
 	if (metaTags["keywords"]) {
 		var keywords = metaTags["keywords"];
 		newItem.tags = keywords.split(",");
-		for (var i in newItem.tags) {
-			newItem.tags[i] = newItem.tags[i].replace("  ", ", ");
+		for (var t in newItem.tags) {
```

Comment:
How about double variable declaration? I.e. something like
```
for (var i=0; ....) {

}
for (var i=0; ...) {

}
```
AFAIK this is still okay, but maybe you have a coding convention on these...